add proper overview and structure (#875)
iotaledger/documentation: client-libraries/1.5/introduction/overview.md

# Client libraries

**The IOTA libraries make it easy for you to integrate IOTA into your own applications. Choose a library to match your use case.**

:::warning:Warning
The libraries are currently in development, and you should not use them in production environments.
:::

## Multi Libraries

> Multi Libraries are libraries written in Rust and bindet to many other languages like C, Python, Node.js, Java, or WebAssembly(WASM).

Reviewer: ```suggestion
Multi Libraries are libraries written in Rust with bindings available in other languages like C, Python, Node.js, Java, or WebAssembly(WASM); Allowing you to utilize this library in the language of your choice.
```

---------------
#### **Client library** ####
[Link](../libraries/client.md)

This library includes core and extended functionality to communicate through an IOTA Node to the Tangle.

- [Rust](../libraries/client.md#rust)
- [Node.js](../libraries/client.md#node.js)
- [Python](../libraries/client.md#python)

---

#### **Wallet library** ####
[Link](../libraries/wallet.md)

The wallet library is a stateful package with a standardised interface for developers to build applications involving IOTA value transactions.

- [Rust](../libraries/client.md)
- [Node.js](../libraries/client.md)

---------------

## More Libraries
### **Official support** ###
---------------

#### **iota.c** ####
[Link](https://github.com/iotaledger/iota.c)

The official C client library for interacting with the Tangle

---

#### **iota.go** ####
[Link](https://github.com/iotaledger/iota.go)

The official Go client library for interacting with the Tangle.

---

#### **iota.js** ####
[Link](https://github.com/iotaledger/iota.js/tree/chrysalis)

Implemeted in TypeScript to strongly type the objects sent and received from the API.

---

---------------

### __Community support__ ###

---------------

> #### __Community Ressources__ ####

Reviewer: ```suggestion
#### __Community Resources__ ####
```

[Link](https://github.com/iota-community)

> Here could appear your library, please feel free to create a Pull Request and add it! :-)

Reviewer: ```suggestion
Here your library could appear, please feel free to create a Pull Request and add it! :-)
```
